pkg/engine/diff.go

```diff
@@ -69,7 +69,6 @@ func getIndentationString(indent int, op deploy.StepOp, prefix bool) string {
 	}
 
 	if result == "" {
-		contract.Assertf(!prefix, "Expected indention for a prefixed line")
 		return result
 	}
 
```

Comment:
i looked at this method, and while this assert seems like it can eb removed, things still feel a bit weird.  The caller is specifically asking for a prefix to be printed.  However, because they are at indent 0 they won't see any prefix at all.

It seems like a potentially better fix would be to update hte code that prints out outputs at the end to ask for things at indent '1' so taht at least the prefix can be supplied.